src/blockdata/transaction.rs

```diff
@@ -765,6 +833,18 @@ impl Decodable for TxIn {
     }
 }
 
+impl Encodable for Sequence {
+    fn consensus_encode<W: io::Write + ?Sized>(&self, w: &mut W) -> Result<usize, io::Error> {
+        self.0.consensus_encode(w)
+    }
+}
+
+impl Decodable for Sequence {
+    fn consensus_decode<R: io::Read + ?Sized>(r: &mut R) -> Result<Self, encode::Error> {
+        Ok(Sequence(Decodable::consensus_decode(r)?))
```

Comment:
I think `Decodable::consensus_decode(r).map(Sequence)` is a bit more readable, not sure.

Reply:
+1

Reply:
Did not know this could be done. Looks much better.